sirepo/package_data/static/js/activait.js

```diff
@@ -66,7 +66,7 @@ SIREPO.app.factory('activaitService', function(appState, panelState) {
         const report = appState.clone(parent);
         const subreports = self.getSubreports();
         report.id = subreports.length
-            ? (Math.max.apply(null, subreports) + 1)
+            ? (utilities.arrayMax(subreports) + 1)
```

Comment:
I kept arrayMin() and arrayMax() in angular because it was easier to write a test for it. In cases where the code was already in angular, I used "utilities".